src/Shared/Resources/Strings.shared.resx

```diff
@@ -278,10 +278,15 @@
     <value>MSB5028: Solution filter file at "{0}" includes project "{1}" that is not in the solution file at "{2}".</value>
     <comment>{StrBegin="MSB5028: "}UE: The project filename is provided separately to loggers.</comment>
   </data>
+  <data name="WildcardResultsInDriveEnumeration" xml:space="preserve">
+    <value>MSB5029: The value "{0}" of the "{1}" attribute in element &lt;{2}&gt; is an invalid wildcard that results in drive enumeration.</value>
```

Comment:
I do not think that a developer reading this will necessarily understand what drive enumeration is or why it is bad. Perhaps something like "... that will result in enumerating all files on the drive, which was likely not intended."